client/my-sites/plans-features-main/hooks/use-is-custom-domain-allowed-on-free-plan.ts

```diff
@@ -0,0 +1,14 @@
+import config from '@automattic/calypso-config';
+import { getTld } from 'calypso/lib/domains';
+
+// TODO
+// it's not really a hook now, but it eventually will contain a request fetching assigned variations from ExPlat endpoints
```

Comment:
Probably remove this comment. In what way is this _not_ a hook?

Reply:
Good point. Initially I added that comment since it's just a simple function that didn't do anything that I'd expect from a hook, e.g. side effect computation, state management, etc. However, it is treated as a hook so this comment is meaningless. It is now removed by https://github.com/Automattic/wp-calypso/pull/78825/commits/b2fb1f2363b6581c6485959b2307b7fd2441831a .